test/performance/odp_l2fwd.c

```diff
@@ -2091,14 +2315,18 @@ static void usage(char *progname)
 	       "                                 1: Create a pool per interface\n"
 	       "  -n, --num_pkt <num>            Number of packets per pool. Default is 16k or\n"
 	       "                                 the maximum capability. Use 0 for the default.\n"
-	       "  -u, --vector_mode              Enable vector mode.\n"
+	       "  -u, --vector_mode <num>        Enable vector mode.\n"
```

Comment:
I think it would be more convenient (e.g. in scripts) if --vector_mode had 3 values, one of them being vector_mode disabled.

Reply:
Updated in v4.